src/app/main/main.c

```diff
@@ -294,6 +294,17 @@ process_signal(int sig)
   }
 }
 
+#ifdef _WIN32
+/** Activate SIGINT on reciving a control signal in console */
+static BOOL WINAPI process_win32_console_ctrl(DWORD ctrl_type) {
```

Comment:
Looks like CI is failing because of the whitespace here. Usually we keep the type on a line on its own, so if you reformat this to:

```
static BOOL WINAPI
process_win32_console_ctrl(DWORD ctrl_type) {
```

Then I think `make check-spaces` will be happy again :-)

Reply:
Ah! I didn't even pay attention to this convention, I guess old habits die hard :)